_posts/2020-11-09-defensive-programing.md

```diff
@@ -0,0 +1,110 @@
+---
+layout: post
+title:  "Defensive programming - what it is and why it is important"
+date:   2020-11-09 12:54:04 -0700
+categories: defensive programming, assertions, pre-conditions, nulls
+---
+# **Defensive programming - what it is and why it is important**
+Gabrielly Lunkes  
+
+![Dwight Schrute from The Office show saying "HE PROTEC"]({{site.url }}https://philippegroarke.com/img/2018/defensive.png)
```

Comment:
You don't need the site.url part here, since you are linking to an external image URL. Just the standard markdown image syntax is enough. You only need the site.url part if you are uploading your own image.